docs/InstrumentationEvents.md

```diff
@@ -29,149 +29,40 @@ Instrumentation Event:
+| CONNECT             |                                                                                                                                                                                                               | Consumer connected with broker. |
+| GROUP_JOIN          | {`groupId`, `memberId`, `leaderId`, `isLeader`, `memberAssignment`, `groupProtocol`, `duration`}                                                                                                                   | Consumer has joined the group. |
+| FETCH_START         | {}                                                                                                                                                                                                            | Starting to fetch messages from broker. |
+| FETCH               | {`numberOfBatches`, `duration`}                                                                                                                                                                           | Fetched messages from the broker. |
+| START_BATCH_PROCESS | {`topic`, `partition`, `highWatermark`, `offsetLag`, `offsetLagLow`, `batchSize`, `firstOffset`, `lastOffset`}                                                                                | Starting to process the batch. |
+| END_BATCH_PROCESS   | {`topic`, `partition`, `highWatermark`, `offsetLag`, `offsetLagLow`, `batchSize`, `firstOffset`, `lastOffset`, `duration`}                                                                  | Processed the batch. |
+| COMMIT_OFFSETS      | {`groupId`, `memberId`, `groupGenerationId`, `topics`}                                                                                                                                                | Committed resolved offsets. |
+| STOP                |                                                                                                                                                                                                               | Consumer has stopped. |
+| DISCONNECT          |                                                                                                                                                                                                               | Consumer has disconnected. |
+| CRASH               | {`error`, `groupId`}                                                                                                                                                                                      | Consumer has crashed. </br> In the case of CRASH, the consumer will try to restart itself. If the error is </br> not retriable, the consumer will instead stop and exit.If your application </br> wants to react to the error, such as by cleanly shutting down resources,</br>restarting the consumer itself, or exiting the process entirely, it should listen </br> to the CRASH event. |
```

Comment:
Are the manual linebreaks here really necessary? The table cell should have a max width with automatic line breaks. This will look weird on mobile or in smaller window sizes.

Reply:
It's not required.
I just wanted for the column to have a max-width, otherwise, it keeps on increasing.
But yeah. I can remove it.

Reply:
We can do that in CSS instead, so it works across device sizes. I can fix it in a separate PR if you remove the linebreaks.

Reply:
Done. Thanks.